test/functional/server/server_spec.lua

```diff
@@ -17,8 +17,8 @@ describe('serverstart(), serverstop()', function()
     eq('', eval('$NVIM_LISTEN_ADDRESS'))
   end)
 
-  it([[sets v:servername _only_ on nvim startup
-      (unless all servers are stopped)]], function()
+  it('sets v:servername _only_ on nvim startup unless all servers are stopped)',
```

Comment:
Changes to this file must not be part of the commit named “Windows: Fix api/vim_spec.lua functional test”.